modules/apps/change-tracking/change-tracking-web/src/main/java/com/liferay/change/tracking/web/internal/display/context/ViewChangesDisplayContext.java

```diff
@@ -217,6 +217,27 @@ public List<FDSActionDropdownItem> getFDSActionDropdownItems() {
 				_language.get(_httpServletRequest, "review-change"), "get",
 				"get", null));
 
+		if (_ctCollection.getStatus() == WorkflowConstants.STATUS_EXPIRED) {
```

Comment:
Can you rewrite the if check to also include drafts or expired so we don't have the move changes logic twice